src/Internal/EventStorePersistentSubscription.php

```diff
@@ -144,7 +145,11 @@ public function start(): void
             $subscriptionDropped
         );
 
-        $operation = $subscription->operation();
+        // @todo dirty hack
+        $property = new ReflectionProperty($subscription, 'subscriptionOperation');
+        $property->setAccessible(true);
```

Comment:
yeah this is quite dirty indeed... do you want to discuss it now or is it something you want to look into later?

Reply:
A possible solution would be to extend this class in order to gain access to this property with a new added accessor, on the other hand I don't want to expose the operation to the end user. So currently I don't know what's best, but at least it's a working (maybe temporary) solution.